azureLogIngestion/adapter/index.ts

```diff
@@ -49,11 +50,20 @@ export default class Adapter {
      * There may be situations where a message corresponds to more than one
      * type of telemetry. In this case, the switch/case may not make sense.
      */
-    processMessages(messages: string, context: Context): void {
-        let records: Records
-
+    processMessages(messages: string | string[], context: Context): void {
         try {
-            records = JSON.parse(messages)
+            const messageArray = _.isArray(messages) ? messages : [messages]
+
+            messageArray.forEach((message) => {
+                const records = JSON.parse(message)
```

Comment:
This appears to have removed JSON parsing error handling?

Reply:
There's still a try/catch wrapped around this...though it's not just the parse. (I could tighten that so that it's just wrapping the parse.)